src/runtime/HtmlController.ts

```diff
@@ -0,0 +1,144 @@
+import {BLOCK_NAME} from '../common';
+import {
+    QueueManager,
+    createQueueWithWait,
+    isHTMLElement,
+    isIFrameLoaded,
+    resizeIframeToFitContent,
+    setIframeStyles,
+} from './utils';
+
+const IFRAME_TAG = 'iframe';
+
+export type HTMLControllerForEachCallback = (
+    block: HtmlDivController | HtmlIFrameController,
```

Comment:
maybe it's worth adding an interface that will be implemented by  HtmlDivController and HtmlIFrameController?

Reply:
deleted HtmlDivController